### ISSUE TITLE
Create new post on 103 responds with a 500

### ISSUE THREAD

**Issue by Skyree** [aaronpk/websub.rocks#25]
After successfully subscribing to a 103 feed, creating a new post results in an http 500.
The item is actually created but the publish action to the sub seems to fail as I don't receive any callback.

### Screenshots
1. The feed contains 3 items
![image](https://user-images.githubusercontent.com/1636904/113433567-b9da1780-93df-11eb-96eb-d77c7564cf5f.png)

2. The publication fails
![image](https://user-images.githubusercontent.com/1636904/113433603-cd857e00-93df-11eb-82fc-4e4243b001e2.png)

3. A new item appears on the feed
![image](https://user-images.githubusercontent.com/1636904/113433628-df672100-93df-11eb-9c64-e4f8aaa49873.png)

**Comment by Alkarex** [comment]
Same for me, also for *102 Atom feed discovery*

**Comment by NotNotQuinn** [comment]
I get the same problem whenever I have successfully subscribed on every subscriber test except 204, because the verification fails (intentionally) and I haven't implemented 101, 102, or 103 so I haven't test those.

To test:
 1. login and go to a random (fresh) blog URL that no one has subscribed to before.
 2. Try to click 'Create New Post', and it should work, if it doesn't try another one, that blog URL likely has a subscription.
 3. Subscribe to the blog URL you used.
 4. Try to click 'Create New Post'. It should just spin, and you shouldn't get a response on the subscription you made.

**Comment by ajorg** [comment]
This bug was caused by an incomplete change at 07ef450 and can be fixed with the following tiny patch (in #29):

```diff
diff --git a/lib/Rocks/Hub.php b/lib/Rocks/Hub.php
index 3de3544..969b92d 100644
--- a/lib/Rocks/Hub.php
+++ b/lib/Rocks/Hub.php
@@ -137,7 +137,7 @@ class Hub {
     );
 
     $route = new \App\Subscriber();
-    $feed = $route->get_feed($request, $response, ['num'=>$num, 'token'=>$token]);
+    $feed = $route->get_feed($request, ['num'=>$num, 'token'=>$token]);
     $html = $feed->getBody();
 
     return (string)$html;
```
